examples/amp-story/bookendv1.json

```diff
@@ -7,8 +7,54 @@
   ],
   "components": [
     {
-      "type": "article-set-title",
-      "title": "test"
+      "type": "heading",
+      "text": "test"
+    },
+    {
+      "type": "small",
+      "title": "This is an example article",
+      "url": "http://example.com/article.html",
+      "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "small",
+      "title": "This is an example article2",
+      "url": "http://example.com/article.html",
+      "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "small",
+      "title": "This is an example article3",
+      "url": "http://example.com/article.html",
+      "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "heading",
+      "text": "test2"
+    },
+    {
+      "type": "small",
+      "title": "This is an example article",
+      "url": "http://example.com/article.html",
+      "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "portrait",
+      "title": "This is an example portrait",
```

Comment:
Per the other comment, this is not actually a title, it's the eyebrow. We should think about what we want to name that, and use that instead (then we can later apply it to the other components as well)

Reply:
how does `category` sound? goes with what the mocks depict and sounds more intuitive

![image](https://user-images.githubusercontent.com/5449100/39883056-c907a458-5453-11e8-8fc5-22f3645dbe4c.png)

![image](https://user-images.githubusercontent.com/5449100/39883037-bc455cf6-5453-11e8-8113-d7853a937caa.png)